CONTRIBUTING.md

```diff
@@ -5,3 +5,7 @@ If you dislike something, please elaborate on why.
 
 If you find something difficult to read, please show examples and why you think
 one version is clearer than another.
+
+This styleguide is generated from the SwiftStyleGuide.playground file. All changes to
+the guide should originate there. Before submitting a PR, please run the ```generate_readme```
+script which will convert the playground into the README.md file used by github.
```

Comment:
Do we need to commit both the playground and the README that the script generates, or does that happen on a build server?